cli/cmd/auth.go

```diff
@@ -91,10 +92,12 @@ keptn auth --skip-namespace-listing # To skip the listing of namespaces and use
 			if err != nil {
 				return err
 			}
+			fmt.Println("Successfully logged out")
```

Comment:
delete this or not?

Reply:
I was requested to add this log, do you mean since it does not work? 😆

Reply:
no, i just thought that might be a "debug fmt" you forgot to delete :D